guides/rules/no-action-cp.md

```diff
@@ -0,0 +1,54 @@
+# No Computed Property Side-effects
+
+In Ember we want data to flow down from the route and send actions up modifiy that data. This means that data flows from the top of the application tree structure and passes down to the child nodes. These children are not the owners of the data passed to them. Instead, they derive their own data based on the passed data. Those children can then choose to pass forward that derived data to its child and so on. If a child needs to mutate data passed to it, it must ask for it by firing any action. Once the owner makes the mutation, that change is propagated back down.
```

Comment:
I wonder if it's worth making it clear that the top-level is the owner of the data. I know you allude to it, but I think it's worth being very specific. 

Thoughts?